frontend/src/store/calendar/actions.ts

```diff
@@ -0,0 +1,23 @@
+import { Dispatch, GetState } from '../types'
+import { ActionTypes, CALENDAR_ACTIONS } from './types'
+
+export const SetIsClicked = (newClickedDate: number) => async (dispatch: Dispatch<ActionTypes>, getState: GetState) => {
+  const { isClicked, clickedDate } = getState().calendar
+  if (isClicked) {
+    if (newClickedDate !== clickedDate) {
+      return
+    } else {
+      dispatch({
+        type: CALENDAR_ACTIONS.SET_IS_CLICKED,
+        newIsClicked: false,
+        newClickedDate: 0,
+      })
+      return
+    }
+  }
```

Comment:
These whole chunk is simply one logic gate

isClicked && newClickDate == clickedDate => dispatch.....

Reply:
Changed in implementation. This code is not used anymore.